kubejs/features.json

```diff
@@ -215,7 +215,7 @@
 	"Remove nether tree fungi": true,
 	"Remove quark saplings": true,
 	"Remove assembly halo and manufactory halo": true,
-	"No More Ingots": false,
```

Comment:
I assume this is not intentional? As I don't see changes here regarding the ingots

Reply:
ah, yeah it isn't